src/widgets/dock_area/mod.rs

```diff
@@ -428,15 +435,16 @@ impl<'tree, Tab> DockArea<'tree, Tab> {
             tabs_ui.set_clip_rect(clip_rect);
 
             // Desired size for tabs in "expanded" mode
-            let expanded_width = available_width / (tabs.len() as f32);
+            let prefered_width =
+                (style.tab_bar.fill_tab_bar).then_some(available_width / (tabs.len() as f32));
```

Comment:
The first pair of parentheses is not necessary (it would be if there was a negation, but there isn't one).